webapp/src/components/Vendor/decentraland/NFTFilters/NFTFilters.tsx

```diff
@@ -383,19 +399,32 @@ const NFTFilters = (props: Props) => {
                   onChange={handleTypeByDropdownChange}
                 />
               </div>
-              <FiltersMenu
-                assetType={assetType}
-                selectedNetwork={network}
-                selectedCollection={contracts[0]}
-                selectedRarities={rarities}
-                selectedGenders={wearableGenders}
-                isOnlySmart={!!onlySmart}
-                onCollectionsChange={handleCollectionsChange}
-                onGendersChange={handleGendersChange}
-                onRaritiesChange={handleRaritiesChange}
-                onNetworkChange={handleNetworkChange}
-                onOnlySmartChange={handleToggleOnlySmart}
-              />
+              {category === NFTCategory.WEARABLE && (
+                <FiltersMenu
+                  assetType={assetType}
+                  selectedNetwork={network}
+                  selectedCollection={contracts[0]}
+                  selectedRarities={rarities}
+                  selectedGenders={wearableGenders}
+                  isOnlySmart={!!onlySmart}
+                  onCollectionsChange={handleCollectionsChange}
+                  onGendersChange={handleGendersChange}
+                  onRaritiesChange={handleRaritiesChange}
+                  onNetworkChange={handleNetworkChange}
+                  onOnlySmartChange={handleToggleOnlySmart}
```

Comment:
Instead of having 2 separate component calls here, I'd just have one and set `undefined` to the props that we don't want.
```suggestion
                  onOnlySmartChange={isWearableCategory ? handleToggleOnlySmart : undefined}
```
same for `onGendersChange`, `selectedGenders`, `onGendersChange` and `isOnlySmart`. 
What do you think?